models/spring-ai-bedrock/src/main/java/org/springframework/ai/bedrock/anthropic/api/AnthropicChatBedrockApi.java

```diff
@@ -109,6 +110,21 @@ public AnthropicChatBedrockApi(String modelId, AwsCredentialsProvider credential
 		super(modelId, credentialsProvider, region, objectMapper, timeout);
 	}
 
+	/**
+	 * Create a new AnthropicChatBedrockApi instance using the provided credentials provider, region and object mapper.
```

Comment:
I think we can also specify here that we require a _new_ parameter, the endpoint override.